src/backend/utils/load/ag_load_labels.c

```diff
@@ -199,15 +202,18 @@ int create_labels_from_csv_file(char *file_path,
 
 
     while ((bytes_read=fread(buf, 1, 1024, fp)) > 0) {
```

Comment:
Brace needs to be on its own line -

some_function_or_command(blah blah,...)
{